bin/views/highlight.html

```diff
@@ -6,6 +6,14 @@
         <link rel=stylesheet href="/assets/styles/monokai.css">
         <script src="/assets/scripts/loc.js" defer></script>
         <script src="/assets/scripts/report.js" defer></script>
+        <script src="/assets/scripts/change-lang.js" defer></script>
+        <noscript>
+            <style>
+                #lang-selection {
+                    display: none;
+                }
+            </style>
+        </noscript>
```

Comment:
Maybe we can rename `report.css` to `hightlight.css` and store that css bit in that renamed file ?

Reply:
note the `<noscript>`

Reply:
I don't understand ?

Reply:
It hide the `select` when the client has JavaScript disabled, because this feature requires JavaScript to work.